src/docs/sdk/client-reports.mdx

```diff
@@ -129,7 +129,8 @@ an envelope item, for example, with the response 429, the client SDK must not re
 this as the server already does. Still, the SDK must record lost envelope items
 when dropping them itself, for example, caused by an active rate limit. SDKs can put
 a simple optional check for HTTP status codes in place where any code `>= 400` except
-`429` will be recorded as `network_error`.
+`429` will be recorded as `network_error`. It is guaranteed that client reports never
```

Comment:
Can it still get rate-limited by ngnix?

Reply:
^ this, exactly. those high-level limits are going to be increased more and more over time since we will try to minimize this from happening.